src/v2/Apps/Partner/Components/PartnerContacts/PartnerContactAddress.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { Text } from "@artsy/palette"
+import { getContactAddressLines } from "./partnerContactUtils"
+import { PartnerContactCard_location } from "v2/__generated__/PartnerContactCard_location.graphql"
```

Comment:
Fragments should always be associated with their respective components.

Reply:
Updated, please take a look.